src/FastBoard.cpp

```diff
@@ -565,6 +581,22 @@ void FastBoard::display_liberties(int lastmove) {
     myprintf("\n\n");
 }
 
+void FastBoard::print_column_labels(int size, std::string padding) {
+	myprintf("   ");
+	for (int i = 0; i < size; i++) {
+		if (i < 25) {
```

Comment:
I think Leela Zero is limited to maximum 25 sized boards so the >= 25 case can probably go.

Reply:
OK. I removed that case.